docs_src/index.md

```
@@ -0,0 +1,184 @@
+- **🔗 Advanced Edge Types**: Supports causal, temporal, correlative, and custom relationship types
+- **📊 Statistical Rigor**: Integrated power analysis and effect size estimation
+- **🎯 Impact-Driven Prioritization**: Focuses on high-impact research directions
+- **🔌 MCP Server**: Seamless Claude Desktop integration with Model Context Protocol
+- **⚡ High-Performance API**: Modern FastAPI implementation with async support
+
+## 🛠️ Technology Stack
+
+<div align="center">
+  <table>
+    <tr>
+      <td align="center"><img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/python/python-original.svg" width="38" height="38"/><br>Python 3.11+</td>
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)